<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F_CONTROLLER</class>
 <widget class="QWidget" name="F_CONTROLLER">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="TW_CONTROLLER">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="usesScrollButtons">
      <bool>true</bool>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <property name="movable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string notr="true">Actions</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Action Histroy:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="TV_ACTION_HISTORY">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="gridStyle">
          <enum>Qt::NoPen</enum>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>2</number>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>2</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>12</number>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>2</number>
         </attribute>
         <attribute name="verticalHeaderMinimumSectionSize">
          <number>12</number>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>2</number>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Run Action from command line:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>-1</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="LE_EDIT_ACTION"/>
             </item>
             <item>
              <widget class="QToolButton" name="TB_ACTION">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>Action</string>
               </property>
               <property name="popupMode">
                <enum>QToolButton::InstantPopup</enum>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
               <property name="arrowType">
                <enum>Qt::NoArrow</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <property name="margin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Status:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="L_ACTION_STATUS">
               <property name="text">
                <string>status</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Usage:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="L_ACTION_USAGE">
               <property name="text">
                <string>usage</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>State Variables</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="margin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>State Variables:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QColumnView" name="CV_STATE_VARIABLES"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Event Log</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="margin">
        <number>5</number>
       </property>
       <item>
        <widget class="QTableView" name="TV_LOG_HISTORY">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
